frontend/src/components/nodes/ValidatorTelemetryRow.tsx

```diff
@@ -61,11 +67,33 @@ const ValidatorTelemetryRow: FC<Props> = ({
                   {producedBlocks !== undefined &&
                   expectedBlocks !== undefined ? (
                     <>
-                      {((producedBlocks / expectedBlocks) * 100).toFixed(3)}%
-                      &nbsp;
-                      <span>
-                        ({producedBlocks}/{expectedBlocks})
-                      </span>
+                      <OverlayTrigger
+                        placement={"bottom"}
+                        overlay={
+                          <Tooltip id="produced-blocks-chunks">
+                            {t(
+                              "component.nodes.ValidatorTelemetryRow.produced_blocks",
+                              {
+                                num_produced_blocks: producedBlocks,
+                                num_expected_blocks: expectedBlocks,
+                              }
+                            )}
+                            {", "}
+                            {t(
+                              "component.nodes.ValidatorTelemetryRow.produced_chunks",
+                              {
+                                num_produced_chunks: producedChunks,
+                                num_expected_chunks: expectedChunks,
+                              }
+                            )}
```

Comment:
I would make it a single translation block `component.nodes.ValidatorTelemetryRow.produced_blocks_and_chunks` since currently it creates this odd combination of two sentences concatenated with a hard-coded comma